### PR TITLE
Use compare_view as backend

### PR DESCRIPTION
**Pull request by christopher-besch**

### CODE REVIEW

**Comment by christopher-besch** [comment]
@kolibril13, could you take a look at this? It's a very rough implementation at the moment.

**Comment by kolibril13** [comment]
Thanks a lot for this contribution! 
It already looks amazing!
I will have a closer look on this soon.

File: jupyter_splitview/inject.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def sanitise_injection(inject: str) -> str:
 def inject_dependencies() -> None:
     js_path = Path(__file__).parents[1] / "vendor/compare_view/browser_compare_view.js"
     # TODO: remove
-    # js_path = Path("/home/chris/compare_view/public/browser_compare_view.js")
+    js_path = Path("/home/chris/compare_view/public/browser_compare_view.js")
```
Comment (psychemedia) [inline jupyter_splitview/inject.py]: Noting that `/home/chris` path is not very generic...

Comment (christopher-besch) [reply jupyter_splitview/inject.py]: Yeah, that's only for debugging. I remove it once this PR is leaving the draft state, which is what the TODO comment is for.

**Comment by christopher-besch** [comment]
@kolibril13, I need your QA skills. This should be ready to deploy once I've written some docs for compare_view. Could you try breaking the current implementation and give me some feedback?

**Comment by kolibril13** [comment]
I found a bug! 🎉 🥲 
When images are not the same dimensions the new backend just shows a white canvas.
The juxtapose.js backend was then aligning the images the following way:
E.g.
```py
%%splity --height auto

img = data.chelsea()
grayscale_img = rgb2gray(img)

fig, ax1 = plt.subplots()
ax1.axis("off")
ax1.imshow(img)
plt.show()

fig, ax2 = plt.subplots()
ax2.axis("off")
ax2.imshow(grayscale_img[:,::4], cmap="gray")
plt.show() 
```

https://user-images.githubusercontent.com/44469195/178704721-375bdb9b-e635-42fd-9c61-2a7e9618cce8.mov

**Comment by christopher-besch** [comment]
Sounds sensible. Could you create an Issue and I'll take care of it after publishing?

**Comment by kolibril13** [comment]
and another bug:

https://user-images.githubusercontent.com/44469195/178712192-fbf49917-d76b-4349-b0da-242ab458bfed.mov

selecting circle

**Comment by christopher-besch** [comment]
I did something about it—could you try it?

**Comment by kolibril13** [comment]
Just tested, still same problem:

https://user-images.githubusercontent.com/44469195/178801563-43bf98ac-5b36-4126-8dc5-9fedd1ecb899.mov

**Comment by christopher-besch** [comment]
What exactly are you doing for this to occur?

**Comment by christopher-besch** [comment]
Please create an Issue for that on compare_view. I'll take a look at this later on.

**Pull request merged**